### ISSUE TITLE
Git spawns thousands of times

### ISSUE THREAD

**Issue by rasa** [ScoopInstaller/Scoop#3877]
On two different systems now, git spawns thousands of times bringing the system to a crawl:
```bat
c:>tasklist | grep git.exe | wc -l
  14279
```

Perhaps the issue is that scoop's `apps/shims/git.exe` runs `apps/git/current/cmd/git.exe`, but instead of it running `apps/git/current/mingw64/libexec/git-core/git.exe` it runs `apps/shims/git.exe`  again, in a vicious cycle?

Any idea what exactly the issue is, and how to fix it?

**Comment by rasa** [comment]
So, when I typed:
```
cd apps\git\2.25.1.windows.1\cmd
set path=
git.exe --version
```
it never returns. Instead, this happens:
![2020-03-14_08-39-04AMprocexp64-Process_Explorer_-_Sysinternals_www sysinternals-cropped](https://user-images.githubusercontent.com/220772/76685311-e9996000-65cf-11ea-82d9-387e9fd56571.png)
Running
```
call scoop uninstall -g git
call scoop install -g git
```
fixed it for now.

**Comment by rasa** [comment]
This just happened with aria2c. For some reason, `C:\apps\shims\aria2c.shim` contains:
```
path = C:\apps\shims\aria2c.exe
```
and `scoop reset aria2` doesn't fix it. wtf?

**Comment by rashil2000** [comment]
Merged in develop branch - #4543
